### PR TITLE
Fix title bar text on save

### LINKED ISSUE #462: Title bar does not update on save
Might be related to #330 but saving a file does not update the file name in the title bar.

nteract-save-as

### PR DESCRIPTION
**Pull request by captainsafia**
Fixes #462.

### CODE REVIEW

**Comment by captainsafia** [comment]
TRRRAAAAAAVVVIIIISSSSSSS.

File: src/notebook/menu/index.js [hunk 1]
```diff
@@ -29,8 +31,9 @@ export function triggerSaveAs(store, dispatch) {
       if (!filename) {
         return;
       }
-      const { notebook } = store.getState();
+      const { notebook, executionState } = store.getState();
       dispatch(saveAs(filename, notebook));
+      BrowserWindow.getFocusedWindw().setTitle(`${tildify(filename)} - ${executionState}`);
```
Comment (rgbkrk) [inline src/notebook/menu/index.js:36]: This makes me wonder if we should be submitting an action for name change.

Comment (captainsafia) [inline src/notebook/menu/index.js:36]: I think that's a good idea. I figure changing the window title is something we'll have to do quite often.

File: src/notebook/menu/index.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export function triggerSaveAs(store, dispatch) {
       if (!filename) {
         return;
       }
-      const { notebook, executionState} = store.getState();
+      const { notebook, executionState } = store.getState();
```
Comment (rgbkrk) [inline src/notebook/menu/index.js:34]: This is a peace offering to the rest of us using `eslint` and wanting a consistent style. :wink:

```
npm run lint
```

If you're in Atom, I recommend installing package `linter-eslint`

Comment (captainsafia) [inline src/notebook/menu/index.js:34]: Yeah, I should probably add it as a pre-commit hook or something.

**Pull request merged**